errbot/__init__.py

```diff
@@ -440,7 +440,7 @@ def a_webhook(self, payload):
 
     if isinstance(args[0], str):  # first param is uri_rule.
         return lambda func: _tag_webhook(func,
-                                         args[0].rstrip('/'),  # trailing / is also be stripped on incoming.
+                                         args[0] if args[0] == '/' else args[0].rstrip('/'),  # trailing / is also be stripped on incoming.
```

Comment:
This line is, unfortunately, too long in its current form:

`errbot/__init__.py:443:121: E501 line too long (139 > 120 characters)`

(https://travis-ci.org/errbotio/errbot/jobs/232904774)